python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -188,22 +196,22 @@ class Job(GangaObject):
     Datasets are highly application and virtual organisation specific.
     """
 
-    _schema = Schema(Version(1, 6), {'inputsandbox': FileItem(defvalue=[], typelist=['str', 'Ganga.GPIDev.Lib.File.File.File'], sequence=1, doc="list of File objects shipped to the worker node "),
-                                     'outputsandbox': SimpleItem(defvalue=[], typelist=['str'], sequence=1, copyable=_outputfieldCopyable(), doc="list of filenames or patterns shipped from the worker node"),
+    _schema = Schema(Version(1, 6), {'inputsandbox': FileItem(defvalue=[], typelist=[str, File], sequence=1, doc="list of File objects shipped to the worker node "),
```

Comment:
This `typelist` and others in this file (with type `FileItem`, ``GangaFileItem` and `ComponentItem`).